addon/components/listbox.js

```diff
@@ -223,6 +246,13 @@ export default class ListboxComponent extends Component {
     }
   }
 
+  scrollIntoView(optionElement) {
+    optionElement.parentElement.scroll(
+      0,
+      optionElement.offsetTop - optionElement.parentElement.offsetTop
+    );
```

Comment:
It seems like `optionElement.scrollIntoView()` has enough browser support that we could call that and simplify this code a bit

https://caniuse.com/?search=scrollIntoView

Thoughts?

Reply:
Aha! `scrollIntoView` was my first go-to, but it ultimately let me down. I've just popped this clarifying comment in the handler:

>    // Cannot use optionElement.scrollIntoView() here because that function
>    // also scrolls the *window* by some amount. Here, we don't want to
>    // jerk the window, we just want to make the the option element visible
>    // inside its container.

Reply:
That's a great caveat and a great comment! Thanks for adding that; I could absolutely see myself trying to refactor this sometime to use `.scrollIntoView` and getting trolled by the difference there.